src/Core/Managed/Shared/Extensibility/Metrics/DependencyMetricExtractor.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+
+using Microsoft.ApplicationInsights.Channel;
+using Microsoft.ApplicationInsights.DataContracts;
+using System.Collections.Generic;
+
+namespace Microsoft.ApplicationInsights.Extensibility.Metrics
+{
+    public class DependencyMetricExtractor : MetricExtractorTelemetryProcessorBase
+    {
+        private const string Version = "1.0";
+
+        private Metric _dependencyCallsDurationMetric;
```

Comment:
Correct, the convention is this.var, no underscores.
StyleCop is your friend (even if you don't like what it tells you to do ;) ).